bundles/org.openhab.binding.cp750/README.md

```diff
@@ -0,0 +1,48 @@
+
+### `sample` Thing Configuration
+
+| Name              | Type    | Description                                       | Default | Required | Advanced |
+|-------------------|---------|---------------------------------------------------|---------|----------|----------|
+| hostname          | text    | Hostname or IP address of the device              | N/A     | yes      | no       |
+| port              | integer | TCP port if different from 61408                  | 61408   | no       | no       |
+| refreshInterval   | integer | Interval the device is polled in sec.             | 5       | no       | no       |
+| reconnectInterval | integer | Interval a new connection is tried after IO error | 10      | no       | no       |
+
+## Channels
+
+_Here you should provide information about available channel types, what their meaning is and how they can be used._
+
+_Note that it is planned to generate some part of this based on the XML files within ```src/main/resources/OH-INF/thing``` of your binding._
+
+| Channel | Type   | Read/Write | Description                                                                                                     |
+|---------|--------|------------|-----------------------------------------------------------------------------------------------------------------|
+| version | Switch | R          | Version as reported by the device                                                                               |
+| fader   | Dimmer | RW         | Fader value (0 to 100)                                                                                          | 
+| mute    | Switch | RW         | Mute (ON or OFF)                                                                                                |
+| input   | String | RW         | Input channel as string (one of "analog", "dig-1", "dig-2", "dig-3", "dig-4", "non-sync" or "mic")              |
+| analog  | Switch | RW         | Is ON if input mode is 'analog'. When an ON command is retrieved, input mode will be changed to this channel.   |
+| dig1    | Switch | RW         | Is ON if input mode is 'dig-1'. When an ON command is retrieved, input mode will be changed to this channel.    |
+| dig2    | Switch | RW         | Is ON if input mode is 'dig-2'. When an ON command is retrieved, input mode will be changed to this channel.    |
+| dig3    | Switch | RW         | Is ON if input mode is 'dig-3'. When an ON command is retrieved, input mode will be changed to this channel.    |
+| dig4    | Switch | RW         | Is ON if input mode is 'dig-4'. When an ON command is retrieved, input mode will be changed to this channel.    |
+| nonsync | Switch | RW         | Is ON if input mode is 'non-sync'. When an ON command is retrieved, input mode will be changed to this channel. |
+| mic     | Switch | RW         | Is ON if input mode is 'mic'. When an ON command is retrieved, input mode will be changed to this channel.      |
```

Comment:
Can you provide full example of .things, .items and .sitemap to the end of this file? 
See other bindings for examples on their readme.md

Reply:
> Can you provide full example of .things, .items and .sitemap to the end of this file? See other bindings for examples on their readme.md

I have started to use OpenHab at Version 3. I know that such things as textual thing definition exists but I don't know how to write them. I have looked into the configuration files of my OpenHab installation but I only find a JSON definition of the Thing configuration. Is there a tool to convert this into the classic syntax?

Reply:
The current version is not yet good, did you use mqtt example? channels should not be there:
Example: https://www.openhab.org/addons/bindings/plugwiseha/#plugwiseha-things